frontend/src/components/common/Pagination.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { Box } from "@chakra-ui/react";
+import {
+  Pagination as P,
+  usePagination,
+  PaginationNext,
+  PaginationPage,
+  PaginationPrevious,
+  PaginationContainer,
+  PaginationPageGroup,
+  PaginationSeparator,
+} from "@ajna/pagination";
+
+const outerLimit = 1;
+const innerLimit = 1;
+
+interface PaginationProps {
+  pagesCount: number;
+  currentPage: number;
+  itemsToShow: number;
+  onPageChange: (page: number) => void;
+}
+
+const Pagination = ({
+  pagesCount,
+  currentPage,
+  onPageChange,
+  itemsToShow,
+}: PaginationProps): React.ReactElement => {
+  const { pages } = usePagination({
+    pagesCount,
+    initialState: { currentPage },
+  });
+
+  return (
+    <Box
+      bottom="0"
+      left="0"
+      right="0"
+      display="flex"
+      justifyContent="center"
+      style={{ height: "75px" }}
+    >
+      <P
+        pagesCount={pagesCount}
+        currentPage={currentPage}
+        onPageChange={onPageChange}
+      >
+        <PaginationContainer>
+          {currentPage !== 1 && (
+            <PaginationPrevious
+              size="paginationPrevious"
+              variant="paginationPrevious"
+            >
+              &lt; Previous
+            </PaginationPrevious>
+          )}
+          <PaginationPageGroup isInline align="center">
+            {pages.map((page: number, index: number) => {
+              if (
+                index === 0 ||
+                index === pages.length - 1 ||
+                (index >= currentPage - innerLimit - 1 &&
+                  index <= currentPage + innerLimit - 1)
+              ) {
+                return (
+                  <PaginationPage
+                    size="PaginationStyle"
+                    variant="PaginationStyle"
+                    key={`pagination_page_${page}`}
+                    page={page}
+                    _current={{
+                      bg: "blue.100",
+                      fontFamily: "DM Sans",
+                      borderRadius: "50%",
+                      color: "grey.100",
+                    }}
+                  />
+                );
+              }
+              if (
+                (index === currentPage - innerLimit - 2 &&
+                  currentPage > outerLimit + innerLimit + 1) ||
+                (index === currentPage + innerLimit &&
+                  currentPage < pagesCount - outerLimit - innerLimit)
+              ) {
+                return (
+                  <PaginationSeparator
+                    size="PaginationStyle"
+                    key={`pagination_ellipsis_${index}`}
+                    sx={{
+                      bg: "blue.50",
+                      color: "grey.300",
+                      height: "2.25rem",
+                      width: "2.25rem",
+                      borderRadius: "50%",
```

Comment:
```suggestion
                      opacity: "1 !important",
                      bg: "blue.50",
                      color: "grey.300",
```
for the styling that is already defined in PaginationStyle, you don't need to redefine it (unless it's not applying like for the colours)

also! the reason the colour was lighter is because the button is disabled and applies `opacity: 0.4` - so we can override it!